web/templates/invite_user_modal.hbs

```diff
@@ -40,16 +40,16 @@
                 <option {{#if this.default }}selected{{/if}} name="expires_in" value="{{this.value}}">{{this.description}}</option>
             {{/each}}
         </select>
-        <p id="expires_on"></p>
-        <div id="custom-invite-expiration-time" class="dependent-settings-block">
+        <p class="input-time-valid-till"></p>
```

Comment:
This class name was noted as possibly not being the best here: https://github.com/zulip/zulip/pull/26109#discussion_r1266911669.

I'm thinking perhaps "time-input-formatted-description" for this one and "custom-time-input-formatted-description" for the custom time input `p` element. It's a bit long, but since these are going to be used in multiple modals, it's better to be clear in what they are.